client/src/modules/reports/generate/stock_entry/stock_entry.html

```diff
@@ -39,6 +39,11 @@ <h3 translate>REPORT.STOCK.ENTRY_REPORT</h3>
             required="true">
           </bh-date-interval>
 
+          <bh-currency-select
+            currency-id="ReportConfigCtrl.currency_id"
```

Comment:
I think this needs to be `ReportConfigCtrl.reportDetails.currencyId`.  That way, it matches the saved value, and doesn't reset to the enterprise currency each time.